skills_utils/s3.py

```diff
@@ -37,6 +38,27 @@ def upload(s3_conn, filepath, s3_path):
     key.set_contents_from_filename(filepath)
 
 
+def upload_dict(s3_conn, s3_path, data_to_sync):
+    """Uploads a dict to an s3 directory, overwriting keys that already exist
```

Comment:
This docstring seems a little misleading; I was guessing the function would create one key with all the data in the dict.  I'd suggest the following changes:

* `s3_path` -> `s3_prefix`
* `"Syncs a dictionary to an S3 bucket, serializing each value in the dictionary as a JSON file with the key as its name."`